### PR TITLE
Fix/presentation name

### PR DESCRIPTION
**Pull request by RV-Adrian**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -64,6 +64,8 @@ var unitTestFiles = [
   "test/unit/**/*.tests.js"
 ];
 
+var commonStyleLink = fs.realpathSync('web/bower_components/rv-common-style') + '/**/*.js';
```
Comment (RV-Adrian) [inline gulpfile.js]: This is an 'improvement' I was going to clarify at retrospection today

**Comment by RV-Adrian** [comment]
@alex-deaconu @fjvallarino 
PR for https://trello.com/c/SFYI1ELt/2157-issue-374-when-a-presentation-name-is-too-long-the-editor-becomes-less-usable-3 issue fix

**Comment by ezequielc** [comment]
@RV-Adrian 👍

**Comment by alex-deaconu** [comment]
@RV-Adrian PR looks good 👍 . However, it needs a rebase at this point, and also was wondering if you could re-stage to another stage. I'm currently staging the prototype branch https://github.com/Rise-Vision/rise-vision-apps/compare/feature/embedded-presentations for my current card on stage-4. Thanks!

**Comment by RV-Adrian** [comment]
@alex-deaconu Sorry about that. I'll take stage-1 and rebase. Thx!

**Pull request merged**